package.json

```diff
@@ -260,7 +260,7 @@
             "es5",
             "all"
           ],
-          "default": "es5",
+          "default": "all",
```

Comment:
@sosukesuzuki, @ntotten, I realize the upgrade to v3 is currently blocked on this extension.
However, since Prettier v3 was released almost a year ago, how would you feel about proactively setting this default to `all` (even with the current v2 support) to be future-proof?
I'm happy to open a PR if it helps :)